src/components/BulkEnrollmentResultsDownloadPage/index.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useContext, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Redirect, useParams } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { logError } from '@edx/frontend-platform/logging';
+import { camelCaseObject } from '@edx/frontend-platform';
+import { ToastsContext } from '../Toasts';
+import EnterpriseAppSkeleton from '../EnterpriseApp/EnterpriseAppSkeleton';
+import LicenseManagerApiService from '../../data/services/LicenseManagerAPIService';
+
+const BulkEnrollmentResultsDownloadPage = ({ enterpriseId }) => {
+  const { enterpriseSlug, bulkEnrollmentJobId } = useParams();
+  const { addToast } = useContext(ToastsContext);
+  const [isLoading, setLoading] = useState(true);
+  const [redirectUrl, setRedirectUrl] = useState(null);
+  const [error, setError] = useState(null);
+  const [notReady, setNotReady] = useState(false);
+
+  useEffect(() => {
+    if (isLoading) {
```

Comment:
I think if we only want this download to happen once then instead of using isLoading check here, just pass the empty params to useEffect is more idiomatic? https://css-tricks.com/run-useeffect-only-once/